src/Form.jsx

```diff
@@ -36,6 +51,10 @@ class Form extends React.Component {
     return initial;
   }
 
+  componentDidMount() {
+    this.props.onMount(this.state);
```

Comment:
instead of calling `onMount()`, we can let `onChange` handle this:
```
Object.keys(props.fields).forEach(name => {
  if (typeof fields[name].defaultValue !== "undefined") {
    this.update(name, field, null, this.state[name]);
  }
})
```